PeerioTechnologies/peerio-desktop
# chore: replace config-base.js with app/package.json/"peerio"

The configuration is now in app/package.json under key "peerio"
instead of config-base.js. Additionally, configuration for
release, when build with peerio-release-builder, will apply
overrides from release/json-overrides.json.

### PR Checklist

- [x] I have merged `dev` into my branch before submitting
- [ ] I have checked with either the repo owner or person who previously worked on this feature to ensure correctness
- [x] I have ensured that I am not using any English strings, only `t('translation_keys)`
- [x] The pull request name follows the standard commit guidelines, e.g. `feat(files): filter by file type`

### Relevant user stories or bugs

### Test plan

* Make sure that @peerio/desktop-release-builder is version 6.0.0.
* Make an unpublished release:

If from this PR:

```
export GH_TOKEN=''
peerio-desktop-release --nosign \
                       --tag json-config \
                       --repository dchest/peerio-desktop \
                       --platforms mac \
                       --destination ./test-build
```

If this PR is merged into dev:

```
export GH_TOKEN=''
peerio-desktop-release --nosign \
                       --tag dev \
                       --repository PeerioTechnologies/peerio-desktop \
                       --platforms mac \
                       --destination ./test-build
```


* Launch an app from ./test-build and check that it connects to prod (`wss://icebear.peerio.com`)

------

The repo owner is @anri-asaturov. The responsible QA specialist is @ebilg.

⬇️ **Note to reviewer:** please squash when merging.
